patches/chrome-common-BUILD.gn.patch

```diff
@@ -10,7 +10,15 @@ index de183da8aeca0e2d4205afd0bc991fff1f382597..36713acdeecf01838e982e26709abf10
  }
  
  source_set("ini_parser") {
-@@ -531,6 +532,7 @@ static_library("non_code_constants") {
+@@ -109,6 +110,7 @@ group("common") {
+ static_library("common_lib") {
+   visibility = [
+     ":*",
++    "//brave/renderer/skus:browser_tests",
```

Comment:
what is needed from this in skus browser tests?

Reply:
in any case it should just depend on `//chrome/common` and remove this patch

Reply:
Fixed in [4e8e261](https://github.com/brave/brave-core/pull/13498/commits/4e8e261cef0726e1ea770188748f4778071fcf31)